src/options.h

```diff
@@ -2398,6 +2398,14 @@ nl_before_class;
 extern BoundedOption<unsigned, 0, 16>
 nl_after_class;
 
+// The number of newlines before a namespace definition.
```

Comment:
I wasn't aware namespaces had "definitions"... I wonder if just saying "a namespace" would make sense?

Reply:
Yes, that was copy/paste of class-related options with global replacement. Fixed in d64de4f